cdap-tms/src/main/java/co/cask/cdap/messaging/store/cache/CachingMessageTable.java

```diff
@@ -65,7 +65,7 @@
                       MessageTableCacheProvider cacheProvider, TimeProvider timeProvider) {
     // Half the tx pruning grace period to be the grace period for scanning the message cache.
     // This is to make sure we won't scan for cached entries that might be pruned.
-    this.gracePeriod = cConf.getLong(PRUNE_GRACE_PERIOD, TimeUnit.HOURS.toSeconds(24)) / 2;
+    this.gracePeriod = cConf.getLong(PRUNE_GRACE_PERIOD, TxConstants.TransactionPruning.DEFAULT_PRUNE_GRACE_PERIOD) / 2;
```

Comment:
see my other comment, you can add this back to cdap-defaults, and make it not be documented

Reply:
@anew This has been done in commit https://github.com/caskdata/cdap/pull/9620/commits/f6531ef305b614e3476af2cac601542f2386686a